python/demos/picture of the day viewer/app.py

```diff
@@ -0,0 +1,160 @@
+"""
+    App.py
+    MediaWiki Action API Code Samples
+
+    Fetches the current Wikipedia Picture of the Day (POTD) and displays it on
+    a webpage. Also allows for user to change the date displayed.
+
+    Attributes
+    ----------
+        change(date_object): Alter current date displayed
+        increment(date_object): Next date
+        decrement(date_object): Previous date
+        index(): Render web page with current POTD
+        fetch_potd(day): Fetch data about current picture from POTD endpoint
+        fetch_image_info(file_name): Fetchs image & description url
+
+    MIT License
+"""
+
+#!/usr/bin/python3
+
+from datetime import date, timedelta
+from flask import Flask, render_template, request
+import requests
+
+APP = Flask(__name__)
+SESSION = requests.Session()
+ENDPOINT = "https://en.wikipedia.org/w/api.php"
+_current_date = date.today()
```

Comment:
I went with lowercase because current_date isn't a constant. But I wasn't clear on the conventions for naming something like it (variable in module scope).

Reply:
Updated it to UPPER_CASE